web_m2x_options/static/src/js/form.js

```diff
@@ -336,6 +394,4 @@ openerp.web_m2x_options = function (instance) {
                 return values;
             })
         },
-    });
-};
-
+});
```

Comment:
It would be nice if the `M2ODialog` object was returned in order to allow for inherited modifications